opendatahub-io/data-science-pipelines-operator
# Replace sample iris-dataset pipeline with a v2 one

## The issue resolved by this Pull Request:
Resolves #425 

## Description of your changes:
This change will replace the sample iris-pipeline in DSPO to an IR version with artifacts included.

## Testing instructions
* Run the following commands to build and deploy the testing environment:
```
$ export QUAY_USERNAME=<replace with your quay.io username>
$ export IMAGE=quay.io/${QUAY_USERNAME}/dspo:v2.1 # Make sure you marked this image as public in your quay.io account
$ export OCP_PIPELINE_NS=openshift-pipelines
$ export DSPO_NS=data-science-pipelines-operator-v2
$ make podman-build podman-push IMG=$IMAGE # Make sure you ran "podman login" first
$ make v2deploy IMG=quay.io/rmartine/dspo:v2.1 V2INFRA_NS=$OCP_PIPELINE_NS OPERATOR_NS=$DSPO_NS
$ make deploy IMG=quay.io/rmartine/dspo:v2.1 V2INFRA_NS=$OCP_PIPELINE_NS OPERATOR_NS=$DSPO_NS
$ oc apply -f https://gist.githubusercontent.com/rimolive/4ac6864d344c293d04001b3aa2ab1fd0/raw/b18682a7f2228d838bc940166b291081302347cd/dspa-v2.yaml
```
* Open kfpv2 UI in the browser, there should be a `[Demo] iris-training` pipeline.
* Expand the pipeline, there should be a pipeline version with the same name
* Click on the pipeline, you should see a graph like this:
![Screenshot from 2023-10-30 14-03-57](https://github.com/opendatahub-io/data-science-pipelines-operator/assets/813430/dbc8278f-4ec4-44c6-9bba-4183f9750540)

## Checklist
